apps/mobile/src/features/send-refactor/forms/btc/btc-form.tsx

```diff
@@ -0,0 +1,153 @@
+import { Controller } from 'react-hook-form';
+
+import { AmountField } from '@/features/send-refactor/components/amount-field';
+import { AssetDisplay } from '@/features/send-refactor/components/asset-display';
+import { ErrorMessage } from '@/features/send-refactor/components/error-message';
+import { Numpad } from '@/features/send-refactor/components/numpad';
+import { Recipient } from '@/features/send-refactor/components/recipient';
+import {
+  SendFormContainer,
+  SendFormFooter,
+} from '@/features/send-refactor/components/send-form-layout';
+import { useBtcForm } from '@/features/send-refactor/forms/btc/use-btc-form';
+import { useSendFlowContext } from '@/features/send-refactor/send-flow-provider';
+import { locale } from '@/features/send-refactor/temporary-constants';
+import { type Account } from '@/store/accounts/accounts';
+import { whenInputCurrencyMode } from '@/utils/when-currency-input-mode';
+import { t } from '@lingui/macro';
+
+import {
+  type AverageBitcoinFeeRates,
+  type FiatCurrency,
+  type MarketData,
+  type Money,
+} from '@leather.io/models';
+import { type Utxo } from '@leather.io/query';
+import { BtcAvatarIcon, Button } from '@leather.io/ui/native';
+import { isNumber } from '@leather.io/utils';
+
+const protocol = 'nativeBtc';
```

Comment:
Maybe we can use `CryptoAssetProtocols` from `models` for this? I guess we should have a more shared `symbol = 'BTC'` also